README.md

```diff
@@ -5,9 +5,17 @@ AMQPPool manages a pool of AMQP channels for you.
 ## Usage
 
 ```elixir
-:ok = AMQPPool.Channel.with_channel(fn channel ->
-  AMQP.Basic.publish(channel, exchange, routing_key, payload)
-end)
+:ok = AMQPPool.Channel.with_channel(:my_pool,
```

Comment:
Should we maybe also provide an overload which uses a "default" pool?

Reply:
We could definitely do that. Could you create a task for it?

Reply:
Sure

Reply:
I did put it directly into the sportsbook backlog, didnt know where to put it else
https://bluelabs-eu.atlassian.net/browse/CLT-55

I also wanted to assing it to a Component but there is no "amqp_pool" component yet and I also have not the permissions to create one